pages/instructorQuestions/instructorQuestions.js

```diff
@@ -34,7 +34,7 @@ router.get('/', function(req, res, next) {
             };
             sqldb.query(sql.questions, params, function(err, result) {
                 if (ERR(err, callback)) return;
-                const ci_ids = _.map(res.locals.course_instances, ci => ci.id);
+                const ci_ids = _.map(res.locals.authz_data.course_instances, ci => ci.id);
```

Comment:
Do we also want `|| []` here?

Reply:
`_.map` accepts nullish as an empty array.

Reply:
cool